keyboards/idobao/id42/info.json

```diff
@@ -0,0 +1,80 @@
+        "device_version": "1.0.0"
+      },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "label": "Esc", "x": 0, "y": 0 },
+                { "matrix": [0, 1], "label": "Q", "x": 1, "y": 0 },
+                { "matrix": [0, 2], "label": "W", "x": 2, "y": 0 },
+                { "matrix": [0, 3], "label": "E", "x": 3, "y": 0 },
+                { "matrix": [0, 4], "label": "R", "x": 4, "y": 0 },
+                { "matrix": [0, 5], "label": "T", "x": 5, "y": 0 },
+                { "matrix": [0, 6], "label": "Y", "x": 6, "y": 0 },
+                { "matrix": [0, 7], "label": "U", "x": 7, "y": 0 },
+                { "matrix": [0, 8], "label": "I", "x": 8, "y": 0 },
+                { "matrix": [0, 9], "label": "O", "x": 9, "y": 0 },
+                { "matrix": [0, 10], "label": "P", "x": 10, "y": 0 },
+                { "matrix": [0, 11], "label": "BSpc", "x": 11, "y": 0 },
+
+                { "matrix": [1, 0], "label": "Tab", "x": 0, "y": 1, "w": 1.25 },
+                { "matrix": [1, 1], "label": "A", "x": 1.25, "y": 1 },
+                { "matrix": [1, 2], "label": "S", "x": 2.25, "y": 1 },
+                { "matrix": [1, 3], "label": "D", "x": 3.25, "y": 1 },
+                { "matrix": [1, 4], "label": "F", "x": 4.25, "y": 1 },
+                { "matrix": [1, 5], "label": "G", "x": 5.25, "y": 1 },
+                { "matrix": [1, 6], "label": "H", "x": 6.25, "y": 1 },
+                { "matrix": [1, 7], "label": "J", "x": 7.25, "y": 1 },
+                { "matrix": [1, 8], "label": "K", "x": 8.25, "y": 1 },
+                { "matrix": [1, 9], "label": "L", "x": 9.25, "y": 1 },
+                { "matrix": [1, 11], "label": "Ent", "x": 10.25, "y": 1, "w": 1.75 },
+
+                { "matrix": [2, 0], "label": "Shft", "x": 0, "y": 2, "w": 1.75 },
+                { "matrix": [2, 1], "label": "Z", "x": 1.75, "y": 2 },
+                { "matrix": [2, 2], "label": "X", "x": 2.75, "y": 2 },
+                { "matrix": [2, 3], "label": "C", "x": 3.75, "y": 2 },
+                { "matrix": [2, 4], "label": "V", "x": 4.75, "y": 2 },
+                { "matrix": [2, 5], "label": "B", "x": 6.75, "y": 2 },
+                { "matrix": [2, 6], "label": "N", "x": 6.75, "y": 2 },
+                { "matrix": [2, 7], "label": "M", "x": 7.75, "y": 2 },
+                { "matrix": [2, 8], "label": "Shft", "x": 8.75, "y": 2, "w": 1.25 },
+                { "matrix": [2, 10], "label": "\u21E7", "x": 10, "y": 2 },
+                { "matrix": [2, 11], "label": "Del", "x": 11, "y": 2 },
+
+                { "matrix": [3, 0], "label": "Ctrl", "x": 0, "y": 3, "w": 1.25 },
+                { "matrix": [3, 1], "label": "Win", "x": 1.25, "y": 3 },
+                { "matrix": [3, 2], "label": "Alt", "x": 2.25, "y": 3, "w": 1.25 },
+                { "matrix": [3, 3], "label": "", "x": 3, "y": 3.5, "w": 2.75 },
+                { "matrix": [3, 6], "label": "Fn", "x": 4, "y": 5.75, "w": 2.75 },
```

Comment:
```suggestion
                { "matrix": [3, 3], "label": "", "x": 3.5, "y": 3, "w": 2.75 },
                { "matrix": [3, 6], "label": "Fn", "x": 6.25, "y": 5.75, "w": 2.75 },
```

Reply:
✓
*(no idea where my mind was when I did that)*